app/src/main/java/com/example/parkingsystemkotlin/mvp/view/DialogFragmentDateTimePickerView.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.parkingsystemkotlin.mvp.view
+
+import com.example.parkingsystemkotlin.fragment.DialogFragmentDateTimePicker
+import com.example.parkingsystemkotlin.mvp.contract.DialogFragmentDateTimePickerContract
+import com.example.parkingsystemkotlin.mvp.view.base.FragmentView
+
+class DialogFragmentDateTimePickerView(dialogFragment: DialogFragmentDateTimePicker) : FragmentView(dialogFragment),
+    DialogFragmentDateTimePickerContract.ViewContract {
+
+    override fun dismissDialogFragment() {
+        (fragment as DialogFragmentDateTimePicker).dismiss()
```

Comment:
instead of DialogFragmentDateTimePicker this should be DialogFragment

Reply:
done